### PR TITLE
rule(write_etc_common): Ignore writes by etckeeper under /etc/.git/

### PR DESCRIPTION
**Pull request by petterreinholdtsen**
Every time etckeeper update the git history of the content in /etc/, it update files in /etc/.git/.  This trigger a warning from falco about writes in /etc/ for every time the cron job or package update.  This change tell the write_etc_common macro to ignore all writes under /etc/.git/ by a process whos great grandparent is etckeeper.  The parent is 'git' and the grandparent is 50vcs-commit.

/kind bug
/kind design
/kind feature
/area rules

Signed-off-by: Petter Reinholdtsen <pere@hungry.com>

Setting WIP to get feedback on the approach, in case there is a better way to do this.  I want to create a similar pull request for cups and /etc/cups/printers.conf, and want feedback on the best alternative.

### CODE REVIEW

**Comment by darryk10** [comment]
Hi @petterreinholdtsen, thanks for the contribution.
Checking the etckeeper official documetation https://wiki.archlinux.org/title/etckeeper, /etc/.git seems to be the default place for it and binding this with `proc.name` and `proc.panme` seems to be the better and safer option to remove such noise.
I'm not sure of adding another macro `etckeeper_activities` for this mentioned in your PR. The problem that I see is that if we start having a macro for each use case, the overall file would be soon full of macros but this is just my feedabck.

**Comment by leogr** [comment]
cc @LucaGuerra

**Comment by leogr** [comment]
Hey @petterreinholdtsen 

Sorry, we lost track of this. We are currently going to release Falco 0.37 in about 2 weeks, then we should have more cycles to look at this.

Meanwhile, let's reopen this
/remove-lifecycle rotten
/reopen

Also, I recommend you to take a look at the new [rules maturity framework](https://github.com/falcosecurity/rules/blob/main/CONTRIBUTING.md#rules-maturity-framework) since I guess this PR needs to be updated to reflect the new way we ship rules.

I apologize again for this to have been taken so long. 

cc @falcosecurity/rules-maintainers

**Comment by petterreinholdtsen** [comment]
It is unclear to me what you want me to do here.  As far as I know, we
are waiting for feedback from one of the developers, and it seem
contra-productive to close the issue after the developers fail to
respond for 90 days.
-- 
Happy hacking
Petter Reinholdtsen

**Comment by incertum** [comment]
@darryk10

**Comment by leogr** [comment]
> It is unclear to me what you want me to do here. As far as I know, we are waiting for feedback from one of the developers, and it seem contra-productive to close the issue after the developers fail to respond for 90 days. -- Happy hacking Petter Reinholdtsen

Hey @petterreinholdtsen 

I apologize again. The `WIP` status was misleading at least for me (I thought this was unfinished). 

re: feedback
> When can feedback from @LucaGuerra be expected? As far as I can tell from the default rules for falco still cause events every time etckeeper update /etc/.git/.

I will ping @LucaGuerra, in case he has lost track of this.

Thank you!

**Comment by LucaGuerra** [comment]
I thought I already approved this. Apologies and thank you for your patience (a lot of it 😅).

**Review by LucaGuerra — APPROVED** [review]

**Comment by petterreinholdtsen** [comment]
[Luca Guerra]
> As soon as the `WIP` is removed from the PR title it will be merged
> automatically.

Done.

-- 
Happy hacking
Petter Reinholdtsen

**Pull request merged**
